flash/image/instance_segmentation/data.py

```diff
@@ -74,15 +177,24 @@ def train_default_transforms(self) -> Optional[Dict[str, Callable]]:
 
 
 class InstanceSegmentationOutputTransform(OutputTransform):
-    @staticmethod
-    def uncollate(batch: Any) -> Any:
-        return batch[DataKeys.PREDS]
+    # @staticmethod
+    # def uncollate(batch: Any) -> Any:
+    #     breakpoint()
+    #     return batch[DataKeys.PREDS]
+    def per_sample_transform(self, sample: Any) -> Any:
+        # TODO GET HERE WITH image SIZE
+        breakpoint()
+        resize = K.geometry.Resize(sample[DataKeys.METADATA]["original_size"][-2:], interpolation="bilinear")
+        sample[DataKeys.PREDS]["mask_array"] = (
+            resize(torch.tensor(sample[DataKeys.PREDS]["mask_array"], dtype=torch.float)) > 0
+        ).to(torch.uint8)
+        # sample[DataKeys.INPUT] = resize(sample[DataKeys.INPUT])
+        return super().per_sample_transform(sample)
 
 
 class InstanceSegmentationData(DataModule):
 
     input_transform_cls = InstanceSegmentationInputTransform
-    output_transform_cls = InstanceSegmentationOutputTransform
 
     @classmethod
     def from_folders(
```

Comment:
Let's also add a `from_numpy` to the datamodule which only has a `predict_array` argument (not train, test, val) :smiley: Actually, should we add this to all icevision tasks???